src/Uno.UI/UI/Xaml/Style/Generic/Generic.Native.xaml

```diff
@@ -342,11 +386,17 @@
 		</Setter>
 	</android:Style>
 
-	<not_wasm:Style TargetType="CommandBar"
-					BasedOn="{StaticResource NativeDefaultCommandBar}" />
+	<wasm:Style
+		x:Key="NativeDefaultCommandBar"
+		TargetType="CommandBar"
+		BasedOn="{StaticResource XamlCommandBar}" />
+
+	<xamarin:Style
+		TargetType="CommandBar"
+		BasedOn="{StaticResource NativeDefaultCommandBar}" />
 
 	<ios:Style x:Key="NativeDefaultAppBarButton"
-			   TargetType="AppBarButton">
+			   TargetType="AppBarButton">ttaskkill
```

Comment:
:D

Reply:
Interesting - this still builds!

Reply:
It's a command or password? ;-)

Reply:
It's our computers that are not fast enough to display the console before I start to type ... wondering who I can bug for that 🤔 😁

But indeed it's strange that the build still succeed ... probably as it's in a style tag, it's just being ignored.